src/AutoRest.CSharp/Common/Output/Models/Requests/PagingResponseInfo.cs

```diff
@@ -31,18 +31,18 @@ public PagingResponseInfo(string? nextLinkName, string? itemName, CSharpType typ
         public CSharpType PageType => new CSharpType(typeof(Page<>), ItemType);
         public CSharpType ItemType { get; }
 
-        private ObjectTypeProperty GetPropertyBySerializedName(CSharpType type, string name)
+        private ObjectTypeProperty GetPropertyBySerializedName(CSharpType type, string name, bool includeParents = false)
```

Comment:
since the only place we are using this **private** method is passing in a `true` for `includeParents`, and I think in this context, we should always include parents - so I think in the future there is never a chance that we will need this new flag.
Therefore I think we do not need to add this, just passing true below.

Reply:
Thanks. Updated